ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_linalg.py

```diff
@@ -328,9 +328,9 @@ def test_kron(
     dtype_x=helpers.dtype_and_values(
         available_dtypes=(ivy.double, ivy.complex64, ivy.complex128),
```

Comment:
This should be `available_dtypes=helpers.get_dtypes("valid")`. The actual supported dtypes will need to be defined using `@with_supported_dtypes`. You can have a look [here](https://lets-unify.ai/ivy/overview/deep_dive/data_types.html#supported-and-unsupported-data-types ) for examples.

Reply:
Hey @AnnaTz, changed as you suggested. ;)

Reply:
```
E           AssertionError:  the results from backend numpy and ground truth framework torch do not match
E            [[-4.54579718e+01 -1.69555650e+03]
E            [ 1.16519772e+14  4.31123155e+15]]!=[[-4.54587648e+01 -1.69558584e+03]
E            [ 1.16519772e+14  4.31123155e+15]] 
E           
E           
E           Falsifying example: test_matrix_exp(
E               dtype_x=(['float64'], [array([[-1.00000000e+00, -1.45519152e-11],
E                        [ 1.00000000e+00,  3.60000000e+01]])]),
E               ground_truth_backend='torch',
E               test_flags=FunctionTestFlags(
E                   num_positional_args=1,
E                   with_out=False,
E                   instance_method=False,
E                   test_gradients=False,
E                   test_compile=None,
E                   as_variable=[False],
E                   native_arrays=[False],
E                   container=[False],
E               ),
E               fn_name='matrix_exp',
E               backend_fw=<module 'ivy.functional.backends.numpy' from '/workspaces/ivy/ivy/functional/backends/numpy/__init__.py'>,
E               on_device='cpu',
E           )
E           
E           You can reproduce this example by temporarily adding @reproduce_failure('6.70.1', b'AXicY2CAAEYkmvGQAaogiInEVoHSACbYAR0=') as a decorator on your test case
```

Hey @AnnaTz , in this NumPy `matrix_exp` function I am getting a mismatching error due to a mismatch in decimal value. How to fix this error any comment on this or should I think of another logic for this?